presto-native-execution/presto_cpp/main/http/HttpClient.h

```diff
@@ -244,7 +244,6 @@ class RequestBuilder {
   send(HttpClient* client, const std::string& body = "", int64_t delayMs = 0) {
     addJwtIfConfigured();
     header(proxygen::HTTP_HEADER_CONTENT_LENGTH, std::to_string(body.size()));
-    headers_.ensureHostHeader();
```

Comment:
I think `send` is used other places too, can this be left in?

Reply:
The problem is this will _always_ set the `Host` header as an empty string because the HttpMessage doesn't have a dstAddress set. It isn't possible to get the dstAddress except for when the client actually sends the request